terraform/modules/oracle_toolkit_module/variables.tf

```diff
@@ -192,3 +209,14 @@ variable "zone" {
   description = "The specific availability zone within the selected GCP region (e.g., us-central1-b)."
   type        = string
 }
+
+variable "assign_public_ip" {
```

Comment:
Now that we use a GCS bucket rather than a github download, can we get away without Internet access at all?